### PR TITLE
Prettier

### PR DESCRIPTION
**Pull request by d-szymanski**
Added prettier to the project to avoid messing style up
also formatted all the files (that may cause conflicts for other PRs)

### CODE REVIEW

**Comment by TobyAsE** [comment]
I believe this should be discussed in #225 before using it. I think prettier is great!

**Comment by d-szymanski** [comment]
> I believe this should be discussed in #225 before using it. I think prettier is great!

Commented there. But this PR doesn't change the style guide. Just adds prettier so it will be easier to follow.

File: package.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "format": "prettier --config .prettierrc \"./**/*.{vue,js}\" --write"
   },
   "dependencies": {
     "bootstrap": "^4.3.1",
     "bootstrap-vue": "^2.0.1",
-    "core-js": "^2.6.5",
```
Comment (alfg) [inline package.json]: Was anything using this? `vue-cli` added this as a dependency when creating the project.

**Comment by alfg** [comment]
Cool, lets try this!

**Pull request merged**